NSS-Day-Cohort-66/rare-api-eager-miracles
# Ndw rareusers tests

### Added Get All and Get One RareUser tests
### Added retrieve method in rare_user ViewSet

### To Test
- Pull down `ndw-rareusers-tests`
- Make sure debugger is running
- In terminal at project level directory, type `python3 manage.py test tests -v 1 `
- Expected Output: `Ran 21 tests in 0.135s
OK
Destroying test database for alias 'default'...`
